index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <title>Portfolio setup and mobile</title>
+</head>
+<body>
+    <header class="toolbar">
+        <a href="#" ><p>My logo </p></a>
+        <div class ="vector">
+            <a href="#" >   <img src = "./Icons/Union.png"> 
+            </a>
+        </div>
```

Comment:
- [ ] As requested by the previous reviewer please wrap this hamburger button with a nav tag for better semantic HTML.